src/utils/validateForm.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
```

Comment:
Only app level utils should be here. Make sure this will be app level validation to be reused in all forms.